src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = !this.hidden;
+
+    return this.hidden;
+  }
 }
 
+Herbivore.prototype.hidden = false;
```

Comment:
As we use classes, let's not use prototypes in this task.
Apply `hidden` property in the corresponding constructor.